account_invoice_merge/wizard/invoice_merge.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2004-2010 Tiny SPRL (http://tiny.be).
+# Copyright 2010-2011 Elico Corp.
+# Copyright 2016 Acsone (https://www.acsone.eu/)
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2019 Okia SPRL
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
```

Comment:
```suggestion
from odoo import api, fields, models
```
Can be replaced by `self.env._`

Reply:
Not mandatory. I'm letting this like that :pray: